WallsAndHoles/mapcell.h

```diff
@@ -19,9 +19,13 @@ class MapCell : public QObject
 
     void setHighlightBrush(const QBrush &color) { mHighlight->setBrush(color); }
 
+    float getTileHeight();
+    void setView(QColor color);
+
 public slots:
     void tileChanged();
-
+signals:
+    void cellUpdated();
```

Comment:
This object shouldn't send any signals out. All it should do is render when told to